core/src/main/java/org/apache/brooklyn/core/entity/EntityAsserts.java

```diff
@@ -223,4 +225,14 @@ public void run() {
         });
     }
 
+    public static void assertServiceFailed(Entity test) {
+        assertFalse(test.sensors().get(SERVICE_UP), "Service should be down");
+        assertEquals(ServiceStateLogic.getExpectedState(test), Lifecycle.ON_FIRE, "Service should be marked on fire");
```

Comment:
Why is this using `getExpectedState`, rather than `assertAttributeEquals(test, Attributes.SERVICE_STATE_ACTUAL, Lifecycle.ON_FIRE)`?